testing/integration-tests/src/full_client/client/unstable_rpcs.rs

```diff
@@ -279,20 +273,26 @@ async fn transaction_unstable_submit_and_watch() {
         .transaction_unstable_submit_and_watch(&tx_bytes)
         .await
         .unwrap();
-
+    println!("about to wait for tx progress");
```

Comment:
nit: we could remove these 2 extra printlns

Reply:
Yup; I completely forgot to go clean up my debug logs by the looks of it! :)